scss/styles.scss

```diff
@@ -2,6 +2,10 @@
   font-family: 'Roboto', sans-serif;
 }
 
+body {
```

Comment:
The body should not have any style applied, define a container at least for the layout. maybe you can change this for a margin top to another component